summary-visualiser/templates/scenarios/dht_sync_lag.html.tmpl

```diff
@@ -68,277 +68,275 @@ and there's no function equivalent to `range` like there is for `template`/`tmpl
     {{ end }}
 {{ end }}
 
-<section class="scenario scenario-dht-sync-lag">
```

Comment:
No changes in this file, except that the guts of the scenario get unwrapped from the `<section>` element -- I moved that to the scenario loop for more predictable HTML structure (and hence more predictable smoke testing and markup reuse).